test/rules/contributing.js

```diff
@@ -0,0 +1,41 @@
+import test from 'ava';
+import m from '../_lint';
+
+const config = {
+	plugins: [
+		require('../../rules/contributing')
+	]
+};
+
+test('contributing - missing', async t => {
+	const messages = await m({config, filename: 'test/fixtures/contributing/error0/readme.md'});
+	t.deepEqual(messages, [
+		{
+			ruleId: 'awesome/contributing',
+			message: 'Missing file contributing.md'
+		}
+	]);
+});
+
+test('contributing - empty', async t => {
+	const messages = await m({config, filename: 'test/fixtures/contributing/error1/readme.md'});
+	t.deepEqual(messages, [
+		{
+			ruleId: 'awesome/contributing',
+			message: 'contributing.md file must not be empty'
+		}
+	]);
+});
+
+// TODO: this test works locally but fails on CI; @sindresorhus any idea why globby is failing here?
```

Comment:
In what way is it failing?

Reply:
It was failing to find the "CONTRIBUTING.md" file on travis-ci which meant that globby wasn't finding the file in a case-insensitive fashion, although it works locally for me on Mac OS.

I'll retry and see if the `case: false` option has solved this issue.